docs/commands.md

```
@@ -1378,6 +1378,31 @@ browse video clean --older-than 24h --dry-run
 
 ## Project Setup
 
+### framework init
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)